lib/metasploit/framework/data_service/remote/managed_remote_data_service.rb

```diff
@@ -1,5 +1,6 @@
 require 'singleton'
 require 'metasploit/framework/data_service/remote/http/core'
+require 'open3'
```

Comment:
Would be good to document what this changes in the PR description or open a separate PR if this is an unrelated change. Just needs to be documented somewhere for logging purposes.